ui/src/tool-panel/location-track/split-store.ts

```diff
@@ -14,7 +14,7 @@ import { getPlanarDistanceUnwrapped } from 'map/layers/utils/layer-utils';
 
 const DUPLICATE_MAX_DISTANCE = 1.0;
 
-type SplitBase = {
+type SplitTargetCandidateBase = {
```

Comment:
Täällä kanssa nimeämisiä uusiksi. Vanha Split-termi muuttui nyt `SplitTargetCandidate`:ksi bäkkärillä käytetyn termin tieltä. Samalla `InitialSplit`:in sijaan nyt puhutaan `FirstSplit`:istä